modules/tinymce/src/themes/silver/main/ts/ui/menus/contextmenu/platform/MobileContextMenu.ts

```diff
@@ -152,11 +153,14 @@ export const initAndShow = (editor: Editor, e: EditorEvent<TouchEvent>, buildMen
       editor.once('touchend', () => editor.off('selectionchange', openiOS));
     }
   } else {
-    // On Android editor.selection hasn't updated yet at this point, so need to do it manually
-    // Without this longpress causes drag-n-drop duplication of code on Android
-    if (isAndroid && !useNodeAnchor) {
-      editor.selection.setCursorLocation(e.target, 0);
-    }
+    // TINY-7688: Commented out cause when longpressing doesn't select and
```

Comment:
Can we avoid merging in commented out code? Either
* Remove it
* Fix it up and uncomment before merging
* Remove it and make a follow-up Jira to fix it, and leave the code in a comment on that Jira as a hint to whoever gets to implement it

Reply:
Did that cause I saw it somewhere hehe. I've removed it.